src/HotChocolate/Data/src/Data/IQueryableExecutable.cs

```diff
@@ -0,0 +1,25 @@
+using System.Linq;
+
+namespace HotChocolate.Data
+{
+    public interface IQueryableExecutable<T> : IExecutable<T>
```

Comment:
Do we need an interface here or is QueryableExecutable enough?

Reply:
Should i remove it?